source/PlayState.hx

```diff
@@ -1,19 +1,131 @@
 package;
 
+import flixel.FlxG;
 import flixel.FlxState;
+import flixel.group.FlxGroup.FlxTypedGroup;
+import flixel.text.FlxText;
 
 class PlayState extends FlxState
 {
+	/** Starting X coordinate of ControllableHero sprite.**/
+	static var CHXPOS:Float = 320;
```

Comment:
No change needed, but I recommend using `CAPITALS_UNDERSCORE_SEPARATED` format for static constant variables. Helps make things quite a bit more readable.

In cases where you have several variables that all about different pieces of a single thing (for example, some are representing X values or Y values) I would make the variable names as identical as possible, and put the part that makes them different at the end to improve readability.

To use this as an example:
`CHXPOS`
`CHYPOS`
becomes
`CH_POS_X`
`CH_POS_Y`

Other things that might make variables easier to keep track of: I generally recommend not abbreviating words within variable names. If a name is too long and cumbersome, there might be another way to represent it, or you might be able to cut the name down to whatever is most essential. If something is very, very commonly abbreviated (like minimum -> min and maximum -> max) those can be okay shortcuts - I'm just leery of everything else.

Related to that, some kinds of information can be implied, like `x` and `y` are typically used to describe the position of something. It's true that things like velocity might also have an x & y component, but the context
 of what class these belong to (Playstate) gives us some idea that it's mostly concerned about where things are located. If you create a pattern of how you name things, you make a standard that makes the whole codebase easier to follow.

So in this case, I'd probably opt for something like
`CONTROLLABLE_HERO_START_X`
`CONTROLLABLE_HERO_START_Y`

By having names like that, I also think I end up eliminating the need add comments above them, because the variable name gives a pretty clear idea that it's about the starting X and Y values for a controllable hero.

Other examples:
```
	static var HERO_START_X:Float = 300;
	static var HERO_START_Y:Float = 300;

	static var WELCOME_TEXT_X:Float = 235;
	static var WELCOME_TEXT_Y:Float = 200;
	static var WELCOME_TEXT_CONTENT:String = "Salvete, omnes!";
	static var WELCOME_TEXT_SIZE:Int = 24;

        static var SCREEN_SAVER_COUNT_MIN = 10;
        static var SCREEN_SAVER_COUNT_MAX = 10;
```